app/views/common_parameters/_parameter.html.erb

```diff
@@ -3,15 +3,15 @@
   <table class="row">
     <tbody>
       <tr class="form-group <%= 'has-error' if f.object.errors.any? %>">
-        <% if controller_name =~ /(host|hostgroup)/ %>
```

Comment:
I think removing it will only cause problems on the host and host group forms where other partials put the "Global" column in for consistency with class parameters, and we probably shouldn't remove that.

Would you mind putting it back, then changing the width of the value column depending on controller_name?  i.e. 4(?) if it's host/hostgroup and 7(?) otherwise?